### ISSUE TITLE
Mismatched documentation/code for set-dimensions

### ISSUE THREAD

**Issue by dwmuller** [spdegabrielle/table-panel#2]
The documentation describes set-dimensions as taking n-rows, then n-cols, but the code actually expects n-cols, then n-rows.

The documented order is consistent with other usages. Unfortunately, fixing the code to match the documentation will be a breaking change.

**Comment by spdegabrielle** [comment]
apologies @dwmuller I only just saw this.

**Comment by spdegabrielle** [comment]
On thinking I think it is better to change the documentation.

**Comment by dwmuller** [comment]
Yeah, the inconsistency is unfortunate, but it's probably better not to break existing code.
